docs/finalproject/intro.md

```diff
@@ -38,20 +38,20 @@ The final projects form the final module of the course, comprising four weeks fo
 Ideally, the project team would consist of the following:
 
 - Four CYF trainees;
-- Three volunteer mentors, forming a [balanced team](https://www.youtube.com/watch?v=Z_Q4Q8rCVpU); and
-- A product owner.
+- Two volunteer mentors ([Tech Lead](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles/tech-lead) and [Product Manager](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles/product-manager), forming a [balanced team](https://www.youtube.com/watch?v=Z_Q4Q8rCVpU); and
+- A [product owner](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles/product-owner).
 
 The mentor group would include an engineer (responsible for unblocking technical/architectural issues, _not_ an extra developer), designer (responsible for user research and UI/UX) and product manager (responsible for prioritisation and alignment with the product owner’s goals). This gives an opportunity to bring in volunteers from the broader tech community, and exposes the trainees to the roles they could be collaborating with in employment.
 
-For more information on roles [click here](./roles.md)
+For more information on roles [click here](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles)
```

Comment:
```suggestion
[More information on volunteer roles](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles)
```

Let's always update these when we see them. Links must make sense out of context. [Success criterion 2.4.4](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html)